fiddler/datadog_checks/fiddler/check.py

```diff
@@ -0,0 +1,142 @@
+# import datetime
+import json
+
+# import yaml
+import time
+from typing import Any
+
+import fiddler as fdl
+
+from datadog_checks.base import AgentCheck
+
+# from datadog_checks.base.utils.db import QueryManager
+# from requests.exceptions import ConnectionError, HTTPError, InvalidURL, Timeout
+# from json import JSONDecodeError
+
+# We will get one hours worth data in chunks and process them.
+bin_size = 3600
+
+# metrics_list = ['accuracy', 'histogram_drift', 'output_average', 'feature_average', 'traffic_count']
+metrics_list = ['accuracy']
+tags = list()
+
+# Connection parameters for Fiddler. Currently hard coded to get some testing done. Will be parameterizing this later.
+URL = 'https://demo.trial.fiddler.ai'
+ORG_ID = 'demo'
+AUTH_TOKEN = 'K4ph7ORDcIO2xVIEA6KxL1o1zHjZockgurhCOZOUSVs'
```

Comment:
Rather than hardcoding these values here, you can put them in your `spec.yaml` and set your values as default values.

Reply:
@yzhan289 - I see that you already answered my earlier question. Will work on spec.yaml. Is there any code snippet that you can share within my check.py where I can read these config values and assign it to these variables?

Reply:
Take a look at our team [integration docs](https://datadoghq.dev/integrations-core/meta/config-specs/) for instructions on how to use `spec.yaml`.